src/main/java/org/sagebionetworks/web/client/widget/header/HeaderViewImpl.java

```diff
@@ -102,22 +102,23 @@ public void setPresenter(Presenter presenter) {
 		refresh();
 	}
 
-	private void detachNavBar() {
-		synapseNavDrawerContainer.removeFromParent();
+	private void hideNavBar() {
+		synapseNavDrawerContainer.addStyleName("hidden");
```

Comment:
I think .setVisible() is preferable, since it will set the display property and the aria-hidden element attribute.